guides/rest_api.md

```diff
@@ -0,0 +1,44 @@
+# REST API
+
+## Authentication
+
+To make full use of the API you need to authenticate as an existing user. A successful authentication returns a *token* that you can use in subsequent requests to authenticate yourself as that user. You do this by setting the token as request header.
+
+> curl -H "Authorization: Bearer AUTH-TOKEN" http://localhost:4000/api/rest/v1/
+
+```
+POST /api/rest/v1/auth
+```
+
+### Parameters
+
+| Name                   | Type     | Description                      |
+| ---------------------- | -------- | -------------------------------- |
+| `audio_file[name]`     | `string` | **Required.** Username or email. |
```

Comment:
`audio_file` needs to be replaced with the proper thing. Nothing?

Reply:
Can you elaborate? I don't follow.

Reply:
The name here, since this is just the authentication point, the username/email should not be audio_file[name], but just name or another basename?

Reply:
🤦‍♂  I took the tomatoes from my eyes, now I see it. Yes of course.